node/src/reactor/validator/memory_metrics.rs

```diff
@@ -0,0 +1,274 @@
+use datasize::DataSize;
+use prometheus::{exponential_buckets, Histogram, HistogramOpts, IntGauge, Registry};
```

Comment:
```suggestion
use prometheus::{self, Histogram, HistogramOpts, IntGauge, Registry};
```

Reply:
That one I don't get - you want me to use `prometheus::exponential_buckets` further down?